cosmwasm/test/injective.sh

```diff
@@ -0,0 +1,30 @@
+# WIP
```

Comment:
This is just a reference file and shouldn't be reviewed, correct?  Because it totally won't work on a fresh chain startup.

Reply:
didn't mean to commit this, thanks for catching